diff -Nru ParseTemplate-0.34/examples/derived.pl ParseTemplate-0.34.new/examples/derived.pl
--- ParseTemplate-0.34/examples/derived.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/derived.pl	Thu Feb  8 07:27:59 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004;
 use strict;
diff -Nru ParseTemplate-0.34/examples/die.pl ParseTemplate-0.34.new/examples/die.pl
--- ParseTemplate-0.34/examples/die.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/die.pl	Thu Feb  8 07:27:36 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004;
 
diff -Nru ParseTemplate-0.34/examples/error.pl ParseTemplate-0.34.new/examples/error.pl
--- ParseTemplate-0.34/examples/error.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/error.pl	Thu Feb  8 07:27:55 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004;
 
diff -Nru ParseTemplate-0.34/examples/html_generator.pl ParseTemplate-0.34.new/examples/html_generator.pl
--- ParseTemplate-0.34/examples/html_generator.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/html_generator.pl	Thu Feb  8 07:27:52 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004; 
 use strict;
diff -Nru ParseTemplate-0.34/examples/html_generator2.pl ParseTemplate-0.34.new/examples/html_generator2.pl
--- ParseTemplate-0.34/examples/html_generator2.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/html_generator2.pl	Thu Feb  8 07:27:31 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 require 5.004; 
 use strict;
 use Parse::Template;
diff -Nru ParseTemplate-0.34/examples/html_template.pl ParseTemplate-0.34.new/examples/html_template.pl
--- ParseTemplate-0.34/examples/html_template.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/html_template.pl	Thu Feb  8 07:27:28 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004; 
 use strict;
diff -Nru ParseTemplate-0.34/examples/includes.pl ParseTemplate-0.34.new/examples/includes.pl
--- ParseTemplate-0.34/examples/includes.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/includes.pl	Thu Feb  8 07:27:48 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # Process a document including directives like this:
 # 
diff -Nru ParseTemplate-0.34/examples/inheritance.pl ParseTemplate-0.34.new/examples/inheritance.pl
--- ParseTemplate-0.34/examples/inheritance.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/inheritance.pl	Thu Feb  8 07:27:24 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004;
 
diff -Nru ParseTemplate-0.34/examples/recursive.pl ParseTemplate-0.34.new/examples/recursive.pl
--- ParseTemplate-0.34/examples/recursive.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/recursive.pl	Thu Feb  8 07:27:44 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004;
 
diff -Nru ParseTemplate-0.34/examples/synopsis.pl ParseTemplate-0.34.new/examples/synopsis.pl
--- ParseTemplate-0.34/examples/synopsis.pl	Wed Sep 27 13:48:42 2000
+++ ParseTemplate-0.34.new/examples/synopsis.pl	Thu Feb  8 07:27:41 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004; 
 use strict;
diff -Nru ParseTemplate-0.34/examples/synopsis_bis.pl ParseTemplate-0.34.new/examples/synopsis_bis.pl
--- ParseTemplate-0.34/examples/synopsis_bis.pl	Mon Sep 25 17:08:57 2000
+++ ParseTemplate-0.34.new/examples/synopsis_bis.pl	Thu Feb  8 07:27:20 2001
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004; 
 use strict;
